decidim-conferences/app/controllers/decidim/conferences/conferences_controller.rb

```diff
@@ -54,6 +54,18 @@ def promoted_conferences
       def stats
         @stats ||= ConferenceStatsPresenter.new(conference: current_participatory_space)
       end
+
+      def conference_participatory_processes
```

Comment:
This logic could be wrapped in cells too. Those cells can render `ParticipatorySpacesCell` directly.

What do you think about this?